v2/pkg/operators/common/dsl/dsl.go

```diff
@@ -923,6 +924,20 @@ func init() {
 
 			return buf.String(), nil
 		}),
+		"ip_format": makeDslFunction(2, func(args ...interface{}) (interface{}, error) {
+			ipFormat, err := strconv.ParseInt(types.ToString(args[1]), 10, 64)
+			if err != nil {
+				return nil, err
+			}
+			if ipFormat <= 0 || ipFormat >= 10 {
+				return nil, fmt.Errorf("invalid value, input must be between 0 and 10")
+			}
```

Comment:
@xm1k3 , I think mapcidr supports 11 formats from [doc](https://github.com/projectdiscovery/mapcidr/wiki/IP-Format-Index) . 

About the error value , I think this is better . What do you think ?
```go
return nil, fmt.Errorf("invalid format, format must be in range 1-11")
```

Reply:
@tarunKoyalwar You are correct, they are 11, not 10. I will correct it now. I will also include the correction you suggested.